src/android/CameraLauncher.java

```diff
@@ -927,6 +859,21 @@ private void writeUncompressedImage(Uri src, Uri dest) throws FileNotFoundExcept
             }
         }
     }
+    /**
+     * In the special case where the default width, height and quality are unchanged
+     * we just write the file out to disk saving the expensive Bitmap.compress function.
+     *
+     * @param src
+     * @throws FileNotFoundException
+     * @throws IOException
+     */
+    private void writeUncompressedImage(Uri src, Uri dest) throws FileNotFoundException,
```

Comment:
I see two writeUncompressedImage functions. Is this correct? The second one is right above this definition.